hr_attendance_reason/i18n/es.po

```diff
@@ -0,0 +1,129 @@
+# Translation of Odoo Server.
```

Comment:
It's better to maintain the translation directly on weblate

Reply:
OK!